django

evcyen opened the pull request:
converted flask to django, added get and post requests at following urls:

http://127.0.0.1:8000/candidates
http://127.0.0.1:8000/vote

evcyen committed:
django

converted flask to django, added get and post requests at following urls:

http://127.0.0.1:8000/candidates
http://127.0.0.1:8000/vote

sophyphreak commented:
Does this use `django-rest-framework` or something similar so this is a fully-functioning API?

evcyen commented:
No, this doesn't - I've never used it but it looks pretty easy to set up from its documentation. I'll work on it and make another commit later

evcyen committed:
added bulk of functionality

added update/detail/delete/list views for users, candidates, and elections. Users able to vote once before being prevented from seeing the page again. Installed django-rest-framework, most pages currently use rest-framework except for custom vote, login, logout, and home views
